src/controllers/resources/form/orders.js

```diff
@@ -300,6 +300,41 @@ export default function () {
       callApi(uri, 'GET', callback, null, skipError)
     }
 
+    // render subscriptions
+    const $listOfSubscriptions = elContainer.find('#t' + tabId + '-invoice-list')
+    if (orderId) {
+      let url
+      const data = Data()
+      if (data.transactions[0].type === 'recurrence') {
+        document.getElementById('subscription-invoices').style.display = 'block'
```

Comment:
ID fixo (quase) nunca, e bota um frufru tbm:

```suggestion
        $(`#${tabId}-subscription-invoices`).slideDown()
```